### PR TITLE
Preparing release 1.1.0

### PR DESCRIPTION
**Pull request by Natim**
r? @leplatrem

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -3,10 +3,16 @@ CHANGELOG
...
 ==================
 
-- Nothing changed yet.
+**Bug fix**
+
+- In case there is a diff using xml-verifier, the command now returns an error code (#28)
+
+**Internal changes**
+
+- ``json2kinto`` is now importing in collection that are not public anymore (#27)
```
Comment (leplatrem) [inline CHANGELOG.rst]: this is not clear.

What about:
```json2kinto`` does not set the destination collection as publicly readable (#27)`
? 

(maybe)

File: tox.ini [hunk 2]
```diff
@@ -11,7 +11,7 @@ install_command = pip install --pre {opts} {packages}
 
 [testenv:functional]
 commands =
-    json2kinto -s http://localhost:8888/v1 --addons-server https://addons.mozilla.org/ -S schemas.json
+    json2kinto -s http://localhost:8888/v1 --addons-server https://addons.allizom.org/ -S schemas.json
```
Comment (leplatrem) [inline tox.ini]: This is the nightly server to have functional tests green?
:+1:

**Comment by leplatrem** [comment]
r+

File: tox.ini (hunk 2 above)
Comment (Natim) [inline tox.ini]: It is the stage env: https://github.com/mozilla/addons-server/#addons-server

**Pull request merged**